src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -140,6 +140,23 @@ internal sealed class TaggerProvider : IViewTaggerProvider, ITableDataSource
         #endregion
 
         public void RequestAnalysis(string path, string charset, IEnumerable<SonarLanguage> detectedLanguages, IIssueConsumer issueConsumer, ProjectItem projectItem)
+        {
+            // Called on the UI thread -> unhandled exceptions will crash VS
```

Comment:
This was another public entry point that could crash VS in the event of errors.

Reply:
can it be tested?

Reply:
Not easily, no. It would require too much refactoring, and I don't want to do it at this point.